ts/util/legend.ts

```diff
@@ -0,0 +1,159 @@
+import { qcStatuses } from "../data/qc-status";
+import { makeIcon } from "./html-utils";
+
+export class Legend {
+  private container: HTMLElement;
+  constructor() {
+    // outer container
+    this.container = document.createElement("div");
+    this.container.className =
+      "text-black block flex flex-col bg-white rounded-xl fixed border-2 border-grey-200 z-40 drop-shadow-lg";
+    this.container.id = "legend-container";
```

Comment:
Make this a constant since it's used in at least 2 places. Will help to avoid bugs if things get renamed/moved